File: src/main/java/com/wikia/tibia/repositories/HuntingPlaceRepository.java
package com.wikia.tibia.repositories;

import com.wikia.tibia.gateways.HuntingPlaceGateway;
import com.wikia.tibia.objects.HuntingPlace;

public class HuntingPlaceRepository extends WikiObjectRepository {

    public HuntingPlaceRepository() {
        super(HuntingPlace.class, new HuntingPlaceGateway());
    }
}

File: src/main/kotlin/com/wikia/tibia/repositories/HuntingPlaceRepository.kt
package com.wikia.tibia.repositories

import com.wikia.tibia.enums.Contract
import com.wikia.tibia.gateways.WikiObjectGateway
import com.wikia.tibia.objects.HuntingPlace

class HuntingPlaceRepository : WikiObjectRepository<HuntingPlace>(
    wikiObjectClass = HuntingPlace::class.java,
    wikiObjectGateway = WikiObjectGateway(Contract.HUNTING_PLACES)
)
